src/main/java/com/google/graphgeckos/dashboard/api/components/Builder.java

```diff
@@ -0,0 +1,35 @@
+package com.google.graphgeckos.dashboard.api.components;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Represents the information about a single buildbot.
+ */
+public class Builder {
+  private final String builderName;
+  private final List<Log> logs;
+
+  // "failed" or "passed"
+  // TODO for mashasamsikova: add java enum representation for these states
+  private final String status;
```

Comment:
I'd suggest defining this enum right now. It does not have to be complete (can contain the "failed" and "passed" states for now). The reason is that changing this from string to an enum later, when different components start using it, would require more work.